interfaces/IDSLs/BodyHandJointsDetector.idsl

```diff
@@ -0,0 +1,25 @@
+module RoboCompBodyHandJointsDetector
+{
+    sequence<byte> ImgType;
+
+    struct TImage {
+        int width;
+        int height;
+        int depth;
+        int num_image;
```

Comment:
Getting error: 
`illegal underscore in identifier `num_image'`

'_' in the variable name is not valid in IDSL. Can you please change this?

Reply:
I have just updated the files, please check them.